portfolioAstro/src/styles/melo.scss

```
@@ -0,0 +1,484 @@
+    display: inline-block;
+    margin-top: 20px;
+    padding: 40px 0;
+    width: 320px;
+    border: $border;
+    text-align: center;
+    cursor: pointer;
+  }
+}
+
+.select-menu-balloon {
+  height: 60px;
```

Stylelint (17.4.0)

[error] Unexpected duplicate "height" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)